recipe/meta.yaml

```diff
@@ -23,12 +25,13 @@ requirements:
     - python
     - pip
     - setuptools
+    - wheel
     - cython
-    - pandas
+    - pandas >=0.16.0
     - six
     - joblib
-    - pyprind
-    - py_stringmatching
+    - pyprind >=2.9.3
+    - py_stringmatching >=0.2.1
```

Comment:
I suspect that:
```
pandas>=0.16.0                                               
six                                                              
joblib                                                           
pyprind>=2.9.3                                                 
py-stringmatching>=0.2.1
```
are only runtime dependencies despite of saying here https://github.com/anhaidgroup/py_stringsimjoin/blob/v0.3.5/README.rst#dependencies.
Look at this file too https://github.com/anhaidgroup/py_stringsimjoin/blob/v0.3.5/docs/installation.rst or https://anhaidgroup.github.io/py_stringsimjoin/v0.3.2/installation.html

Reply:
Yeah, I agree they don't feel right for a build -- this is a fork of conda-forge, of course.

I've chopped them out of `host` and the build/test seems OK.  I'll push that change for a CI run.